bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/net/NetUtil.java

```diff
@@ -80,6 +122,20 @@ public synchronized void modified(Map<String, Object> config) {
         } else {
             configuredBroadcastAddress = broadcastAddressConf;
         }
+
+        String pollFrequenceSecondsStr = "";
+        int pollFrequenceSeconds = POLL_DEFAULT_FREQUENCY_SECONDS;
+        try {
+            Object pollFrequenceSecondsObj = config.get(POLL_INTERVAL);
+            if (pollFrequenceSecondsObj != null) {
+                pollFrequenceSeconds = Integer.parseInt(pollFrequenceSecondsObj.toString());
+            }
+        } catch (NullPointerException | NumberFormatException e) {
+            LOGGER.debug("Cannot parse value {} from key {}, will use default {}", pollFrequenceSecondsStr,
```

Comment:
I'd be also ok with a WARN here as it is an invalid user configuration that he might want to know about.